src/main/java/cstwooneohthree/glados/tasks/TaskList.java

```diff
@@ -0,0 +1,165 @@
+package cstwooneohthree.glados.tasks;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import cstwooneohthree.glados.enums.TaskType;
+import cstwooneohthree.glados.exceptions.GladosException;
+import cstwooneohthree.glados.exceptions.TaskNotFoundException;
+import cstwooneohthree.glados.utils.ParsedInfo;
+import cstwooneohthree.glados.utils.Parser;
+import cstwooneohthree.glados.utils.Storage;
+
+/**
+* TaskList is used to store and manage tasks in GLaDOS.
+*
+* @author jayjay19630
+*/
+public class TaskList {
+
+    /* ArrayList to keep track of all tasks */
+    private ArrayList<Task> tasks;
+    /* Integer to keep track of tasks length */
+    private int listIndex;
```

Comment:
Your variable names clearly distinguish between single-valued and multi-valued variables, which is great!